src/Conversion/ONNXToKrnl/NN/Conv.cpp

```diff
@@ -249,6 +250,18 @@ struct ONNXConvOpLowering : public OpConversionPattern<ONNXConvOp> {
     Value alloc =
         create.mem.alignedAlloc(memRefType, shapeHelper.getOutputDims());
 
+    if (checkOpToCall(op, opsForCall)) {
```

Comment:
I may need more of a big picture, it feels like changing onnx ops to call should be done in a separate pass, not inside each operations that might be converted. What do you think @chentong319 ?

Reply:
The big picture is to break down the current onnx to krnl lowering into several steps:
1. separate out the memref creation into a independent pass as the bufferization pass. The one-shot pass will be used.
2. A pass to lower some onnx ops to calls
3. A pass to lower the rest onnx ops to krnl dialect.

Reply:
Generally agree, and as an initial step, we should not conflate 2 &3 given that 3 is new. Please add a separate pass for this now.